README.md

```diff
@@ -124,10 +172,95 @@ POST http://my-api.com/endpoint
 
 Type `jsonr --help` for more details on usage once you have a tool installed.
 
+### Configuration Files
```

Comment:
Let's remove this section from readme. Instewd elaborate more within help.js, especially I like how we explain recursive config file read and how it is merged